packages/core/src/tools/create-response-stasher.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const _ = require('lodash');
+const uploader = require('./uploader');
+const crypto = require('crypto');
+
+const withRetry = async (fn, retries = 3, delay = 100, attempt = 0) => {
+  try {
+    return await fn();
+  } catch (error) {
+    if (attempt >= retries) {
+      throw error;
+    }
+
+    await new Promise((resolve) => setTimeout(resolve, delay));
+    return withRetry(fn, retries, delay, attempt + 1);
+  }
+};
+
+// responseStasher uploads the data and returns the URL that points to that data.
+const stashResponse = async (input, response, size) => {
+  const rpc = _.get(input, '_zapier.rpc');
+
+  if (!rpc) {
+    throw new Error('rpc is not available');
```

Comment:
Asking for my own curiosity 🧠 here - when/how would rpc be unavailable?

Reply:
Usually it's fine, just sanity check. `create-rpc-client` looks for a few things, esp if the Lambda event doesn't have all the values it needs to create the client